core/state/state_object.go

```diff
@@ -224,13 +226,18 @@ func (s *stateObject) GetCommittedState(key common.Hash) common.Hash {
 	}
 	s.db.StorageLoaded++
 
-	start := time.Now()
+	var start time.Time
+	if metrics.EnabledExpensive() {
```

Comment:
```
	var start time.Time
	if metrics.EnabledExpensive() {
		start = time.Now()
	}
```
can't save more time than 
```
start := time.Now()
```	

so, for simplicity, I think this change can be reverted